### ISSUE TITLE
General Discussion

### ISSUE THREAD

**Issue by ellerychan** [brata-hsdc/brata#98]
This is meant to be a general conversation thread, not an issue that needs resolution.  It is an alternative to sending emails to the large list of contributors that we have.  @jawaad-ahmad suggested this as an alternative when I tried to share [this link about Adafruit](http://theinstitute.ieee.org/technology-focus/technology-topic/how-diy-electronics-startup-adafruit-industries-became-a-multimillion-dollar-company) with everyone (because we are using some of their hardware for the HSDC).

**Comment by mmieczkowski** [comment]
Is the brata still using SMS?

**Comment by ellerychan** [comment]
All the BRATA communication that I am aware of uses HTTP now, so only a data connection is needed.

**Comment by dvjones12** [comment]
That is correct.  We stopped using SMS because it was unreliable with some of the cell networks.

**Comment by mmieczkowski** [comment]
That's great! It won't port to iOS.

**Comment by dvjones12** [comment]
true but I did the Frequency tone detector.  So may need to do that.  however I am trying to work out what the design needs to be with Mike and trying to NOT use android specific DSP libraries.  Hope is that it will be a relatively easy port to iOS

**Comment by ellerychan** [comment]
I just saw [this write-up](https://www.raspberrypi.org/blog/raspbian-jessie-is-here/) of the latest **Raspbian** release based on **Debian Jessie**.

Of particular interest to some of you _brata.station_ developers:

> Python users used to have to launch Python with sudo in order to allow access to the GPIO lines – Python can now access GPIOs as a standard user.
